mllib-dal/src/main/scala/com/intel/oap/mllib/regression/LinearRegressionDALImpl.scala

```diff
@@ -57,11 +57,11 @@ class LinearRegressionDALImpl( val fitIntercept: Boolean,
   /**
    * Creates a [[LinearRegressionDALModel]] from an RDD of [[Vector]]s.
    */
-  def train(labeledPoints: Dataset[_]): LinearRegressionDALModel = {
+  def train(labeledPoints: Dataset[_], featuresCol: String): LinearRegressionDALModel = {
```

Comment:
Could you also add labelCol and fix rddLabeledPointToMergedTables and rddLabeledPointToSparseTables?